### PR TITLE
ocp4: update OCP4 rule selections, update platform tags

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by JAORMX** [comment]
This seems to break the image build:

```
 [  0%] Built target ocp4-tables
[  0%] [bash-remediation-functions] generating bash-remediation-functions.xml
[  0%] Built target generate-internal-bash-remediation-functions.xml
Scanning dependencies of target generate-internal-ocp4-shorthand.xml
[  1%] [ocp4-content] compiling profiles
[  1%] [ocp4-content] generating shorthand.xml
Traceback (most recent call last):
  File "/content/build-scripts/yaml_to_shorthand.py", line 95, in <module>
    main()
  File "/content/build-scripts/yaml_to_shorthand.py", line 82, in main
    p.validate_rules(loader.all_rules, loader.all_groups)
  File "/content/ssg/build_yaml.py", line 307, in validate_rules
    raise ValueError(msg)
ValueError: Rule 'package_crypto-policies_installed' was not found in the benchmark. Please remove rule 'package_crypto-policies_installed' from profile 'moderate' before proceeding.
make[2]: *** [ocp4/CMakeFiles/generate-internal-ocp4-shorthand.xml.dir/build.make:2225: ocp4/shorthand.xml] Error 1
make[1]: *** [CMakeFiles/Makefile2:1035: ocp4/CMakeFiles/generate-internal-ocp4-shorthand.xml.dir/all] Error 2
make: *** [Makefile:168: all] Error 2
error: build error: running './build_product --debug ocp4' failed with exit code 2
2020/04/16 16:44:22 No custom metadata found and prow metadata already exists. Not updating the metadata.
2020/04/16 16:44:23 Ran for 2m16s
2020/04/16 16:44:23 Submitted failure event to sentry (id=9fd1f7094e4549e89f8ef8fc684c6802)
error: could not run steps: step ocp4-content-ds failed: could not wait for build: the build ocp4-content-ds failed after 1m41s with reason DockerBuildFailed: Docker build strategy has failed.
ValueError: Rule 'package_crypto-policies_installed' was n...icies_installed' from profile 'moderate' before proceeding.
make[2]: *** [ocp4/CMakeFiles/generate-internal-ocp4-shorthand.xml.dir/build.make:2225: ocp4/shorthand.xml] Error 1
make[1]: *** [CMakeFiles/Makefile2:1035: ocp4/CMakeFiles/generate-internal-ocp4-shorthand.xml.dir/all] Error 2
make: *** [Makefile:168: all] Error 2 
```

File: ocp4/profiles/moderate.profile [hunk 1]
```diff
@@ -70,7 +70,7 @@ selections:
     #- sshd_disable_gssapi_auth
     #- var_sshd_set_keepalive=0
     # AC-2(5)
-    - sshd_set_keepalive
+    # - sshd_set_keepalive
```
Comment (redhatrises) [inline ocp4/profiles/moderate.profile:73]: @shawndwells why was this disabled?

File: ocp4/profiles/moderate.profile [hunk 2]
```diff
@@ -302,7 +304,7 @@ selections:
     ## AC-11(a) / FMT_MOF_EXT.1
     #- sshd_idle_timeout_value=10_minutes
     # AC-2(5)
-    - sshd_set_idle_timeout
+    #- sshd_set_idle_timeout
```
Comment (redhatrises) [inline ocp4/profiles/moderate.profile:307]: @shawndwells why was this disabled?

File: ocp4/profiles/moderate.profile [hunk 3]
```diff
@@ -230,19 +231,19 @@ selections:
...
     ### Configure SSSD
     - sssd_run_as_sssd_user
 
     ### Configure USBGuard
-    - service_usbguard_enabled
-    - configure_usbguard_auditbackend
-    - usbguard_allow_hid_and_hub
+    #- service_usbguard_enabled
+    #- configure_usbguard_auditbackend
+    #- usbguard_allow_hid_and_hub
```
Comment (redhatrises) [inline ocp4/profiles/moderate.profile:246]: These should already be approved for usage. Don't think that we should remove.

File: ocp4/profiles/moderate.profile [hunk 4]
```diff
@@ -230,19 +231,19 @@ selections:
     - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    - package_fapolicyd_installed
-    - service_fapolicyd_enabled
+    #- package_fapolicyd_installed
+    #- service_fapolicyd_enabled
```
Comment (redhatrises) [inline ocp4/profiles/moderate.profile:235]: This is a vertical and procurement requirement.

File: ocp4/profiles/moderate.profile (hunk 3 above)
Comment (shawndwells) [reply ocp4/profiles/moderate.profile:246]: They can be re-enabled once the packages ship in CoreOS.

File: ocp4/profiles/moderate.profile (hunk 1 above)
Comment (shawndwells) [reply ocp4/profiles/moderate.profile:73]: Other rules disable SSH entirely.

File: ocp4/profiles/moderate.profile (hunk 4 above)
Comment (shawndwells) [reply ocp4/profiles/moderate.profile:235]: Unfortunately, for now, these packages do not ship in CoreOS. Once the packages ship the rules can be re-enabled.

Comment (shawndwells) [reply ocp4/profiles/moderate.profile:235]: eg every CoreOS box would be flagged for missing this package, but it is not available, so seems like unnecessary pain. Tracking of fapolicyd inclusion is done at the product level for now.

**Comment by shawndwells** [comment]
Content builds, OCP e2e throwing errors. Attempting retest.

**Comment by matejak** [comment]
Closing, as there is a long-time inactivity, and a huge number of conflicts. If anybody wishes to get this PR merged, reopen it, and assign yourself.

**Pull request closed without merging**